src/main/resources/application.properties

```diff
@@ -19,11 +19,15 @@ azure.accessTokens.path=
 #    End with letter or digit
 #    Can't contain consecutive hyphens.
 #    3 <= Length <= 24
-helium.keyvault.name=${KeyVaultName}
+helium.keyvault.name=sanjeevdhe
```

Comment:
don't checkin an actual value.  I just leave it as ${KeyVaultName} and on my debug terminal do an "export KeyVaultName=jf2he".

Reply:
This change was already done before the PR went. It says outdated on the app properties, not sure why it showed you the previous version.